.github/workflows/CI.yml

```diff
@@ -56,15 +56,25 @@ jobs:
         restore-keys: |
           ${{ runner.os }}-pip-
 
+    - name: Create and start virtual environment
+      run: |
+        python3 -m venv venv
+        source venv/bin/activate
+
     - name: Install dependencies
       run: |
         pip3 install --user codecov
+        pip3 install -r requirements.txt
```

Comment:
This too? Sure it happens during the docker build in the next step.

Reply:
Also removed, the docker installation was sufficient